tap_s3_csv/sync.py

```diff
@@ -249,15 +249,19 @@ def sync_csv_file(config, file_handle, s3_path, table_spec, stream, timers={}):
             config, source_type_map)
         timers['resolve_fields'] += time.time() - start
 
+        # modify schema to put null as the last time to check for
```

Comment:
"time" -> "type"
put this comment at the top of transform_schema_recur in transform.py so we know what the function is expected to do